### DISCUSSION TITLE
Can i increase the amount of rays per pixel?

### DISCUSSION THREAD

**Discussion by BryanAM7** [sultim-t/xash-rt#151] [Q&A] [answered]
Hi every one and thanks to sultim for this beauty mod.
There is any way or some console command to increase the rays per pixel? to decrease nosi ein some parts? would be nice to try differnt settings for perfomance and quality purposes like # of indirect bounces, rays per pixel etc.

**Comment by sultim-t** [comment, marked as answer]
It's kinda hardcoded... so the main property that affects performance is the render resolution (i.e. tweaking DLSS/FSR2).
There's rt_indir_bounces (can be set to 1 or 2), but <2 bounces for indirect illumination doesn't look plausible.

**Reply by BryanAM7** [reply]
Hey ty for the reply!!! ill check the sdk then. 
Greetings!
